picosystem/src/display.rs

```diff
@@ -71,31 +82,31 @@ impl Display {
         lcd_reset_pin.into_push_pull_output();
         let spi = Spi::<_, _, 8>::new(spi_device).init(
             resets,
-            125_000_000u32.Hz(),
+            125.MHz(),
             62_500_000u32.Hz(),
             &MODE_3,
         );
         let di = SPIInterfaceNoCS::new(spi, lcd_dc_pin);
-        let mut st7789 = ST7789::new(di, lcd_reset_pin, WIDTH as u16, HEIGHT as u16);
+        let mut st7789 = ST7789::new(di, Some(lcd_reset_pin), Some(backlight_pin), WIDTH as u16, HEIGHT as u16);
         st7789.init(delay_source).unwrap();
         st7789.set_tearing_effect(TearingEffect::Vertical).unwrap();
         let mut display = Display {
             st7789,
-            backlight_pin,
             dma_channel,
             lcd_vsync_pin,
             last_vsync_time: 0,
         };
         // A single clear occasionally fails to clear the screen.
         for _ in 0..2 {
-            let colors =
-                core::iter::repeat(RawU16::from(Rgb565::BLACK).into_inner()).take(WIDTH * HEIGHT);
+            // let colors =
```

Comment:
Dead code

Reply:
Well, if you say so, I would suggest we remove it.